### PR TITLE
fix: isend sms provider unexpected provider error

### LINKED ISSUE #5123: 🐛 Bug Report: A new provider isend-sms throws an error but when trying to view the error details, Novu provides an empty object!
### 📜 Description

When tried to send an SMS using the correct credentials, the provider throws an error, I am unable to fix this error because Novu provides an empty object and doesn't provide any useful details about the error!

### 👟 Reproduction steps

1. Setup APNS provider
2. Go to the workflow builder, and create a new workflow that consumes the isend-sms provider
3. Trigger the test from the workflow builder

### 👍 Expected behavior

It should provide an object that includes the details about where and why the error happened. It should also show the error message thrown by the provider.

### 👎 Actual Behavior with Screenshots

When pressed the "Show details" button on the error event it shows an empty object!

image

### Novu version

Novu SaaS

### npm version

_No response_

### node version

_No response_

### 📃 Provide any additional context for the Bug.

_No response_

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find a similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by jainpawan21**
### What change does this PR introduce?
1. Change png image to SVG image
2. Add apiToken in credential schema and dto
3. Add error name and message in execution details log 

### Why was this change needed?

closes #5123

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "commit": "cz",
     "nx": "nx",
     "lint-staged": "lint-staged",
-    "generate:provider": "npx hygen provider new --version 0.16.3",
+    "generate:provider": "npx hygen provider new --version 0.22.0",
```
Comment (BiswaViraj) [inline package.json:16]: Nice 👌🏼 
But I am wondering if there is a way we can automate this version number.

Comment (jainpawan21) [reply package.json:16]: Not sure how to read version in package.json 🙂

Comment (BiswaViraj) [reply package.json:16]: Let's create a linear ticket for that and address it later

**Review by BiswaViraj — COMMENTED** [review]
Please add the provider to the `sort-providers` file on web

File: package.json (hunk 1 above)
Comment (jainpawan21) [reply package.json:16]: https://linear.app/novu/issue/NV-3480

**Comment by jainpawan21** [comment]
> Please add the provider to the `sort-providers` file on web

Added ✅

**Review by BiswaViraj — APPROVED** [review]

**Pull request merged**